bigbluebutton-html5/imports/ui/components/modal/random-user/container.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { withTracker } from 'meteor/react-meteor-data';
+import Meetings from '/imports/api/meetings';
+import Users from '/imports/api/users';
+import Auth from '/imports/ui/services/auth';
+import { withModalMounter } from '/imports/ui/components/modal/service';
+import { makeCall } from '/imports/ui/services/api';
+import RandomUserSelect from './component';
+
+const RandomUserSelectContainer = props => <RandomUserSelect {...props} />;
+
+export default withModalMounter(withTracker(({ mountModal }) => {
+  const viewerPool = Users.find({
```

Comment:
For this code, given that we only need the number of eligible viewers to pick from, we could use .find(..).count()